ThingIFSDK/ThingIFSDK/TimeRange.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  TimeRange.swift
+//  ThingIFSDK
+//
+//  Created on 2017/01/30.
+//  Copyright (c) 2017 Kii. All rights reserved.
+//
+
+import Foundation
+
+open class TimeRange: NSCoding {
+
+    /** Start point of a time range. */
```

Comment:
I agree with mentioning inclusive or not is important but we should mention about it at using oint such as `AggreagedResult.timeRange`. 

I think inclusive or exclusive depends on its use case.

What do you think?